resources/definitions/ultimaker.def.json

```diff
@@ -37,6 +37,11 @@
         "bridge_wall_material_flow": { "value": "wall_material_flow" },
         "bridge_wall_speed": { "value": "bridge_skin_speed" },
         "cool_fan_speed_0": { "value": "cool_fan_speed_min" },
+        "cool_fan_speed_max": { "value": "100" },
+        "cool_min_layer_time": { "value": 6 },
+        "cool_min_layer_time_fan_speed_max": { "value": "cool_min_layer_time + 5" },
+        "cool_min_speed": { "value": "round(speed_wall_0 * 3 / 4) if cool_lift_head else round(speed_wall_0 / 5)" },
```

Comment:
It might be good practice to start using `resolveOrValue('<setting_name>')`, this will ensure that the settings can be serialized correctly. I don't think this particular setting will cause problems, but it is good practice imo
 https://github.com/Ultimaker/Cura/blob/main/docs/profiles/getting_a_setting_value.md#evaluating-a-value-property 

and

https://github.com/Ultimaker/Cura/issues/13675#issuecomment-1367995394